modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -1826,6 +1836,37 @@ m| dbms.cluster.uncordonServer(server :: STRING)
 m|WRITE
 |===
 
+[[procedure_dbms_cluster_switchdiscoveryserviceversion]]
+[role=label--enterprise-edition label--admin-only]
+.dbms.cluster.switchDiscoveryServiceVersion()
+[cols="<15s,<85"]
+|===
+| Description
+a| This procedure allows you to select which discovery service should be started.
```

Comment:
Is it worth adding a note like "Note that transitions must be done in order, which means it is not possible to switch to `V2_OVER_V1` when current server is in `V1_ONLY`, the server must first go be switched to `V1_OVER_V2`

Reply:
I think we don't need to do this because other procedures don't describe their contract in this document(correct me if I'm wrong). Also If a customer tries to do illegal transitions, this procedure will crash and print an informative message about the right transition. You can find what message will be printed here `com.neo4j.causalclustering.discovery.parallel.ParallelTopologyService#switchMode`.